vagrant.yml

```diff
@@ -0,0 +1,13 @@
+---
+box: centos/7
```

Comment:
Can you add a comment stating that when you want to change the box, you need to run `vagrant reload` or it won't be taken into account if you've already run `vagrant up` once.

Careful with `vagrant reload`, if you have already running VM, it can recreate them. (Which is necessary anyway if you change the box)

Reply:
Comment added.